jasmin_manage/serializers/service.py

```diff
@@ -98,3 +115,19 @@ def create(self, validated_data):
         # Inject the project from the context into the model
         validated_data.update(project=self.context["project"])
         return super().create(validated_data)
+
+    def get_is_active_requirements(self, obj):
+        # Works out if there are any active requirements in the service
+        return True if obj.get_num_active_requirements() else False
+
+    def get_consortium(self, obj):
```

Comment:
Possibly (I can't remember) if these three become properties you could just add them to `fields` without the accessor methods. Not quite sure if there are any advantages to that really.